USAGE.md

```diff
@@ -0,0 +1,307 @@
+
+![bundle id on Xcode](http://drive.google.com/uc?export=view&id=0B7UxsiswNc5_Tno4VHA4Snh3alE)
+
+## Capturing image
+
+Before using userdive heatmap, capture the images of app view.
+For capturing image, start USERDIVE iOS SDK with developer mode.
+
+
+### Capture images when it detects view transitions
+
+USERDIVE iOS SDK can automatically tracking user gesture, and view transitions,
+if you use standard iOS UI components (UINavigationController, UITabBarController etc...).
+
+Import USERDIVE iOS SDK header.
+
+```objective-c
+#import "Userdive.h"
+```
+
+Insert `[Userdive startDeveloperMode:<YOUR_TEAM_ID>]` into 
+`AppDeletegate:application:didFinishLaunchingWithOptions:` with your Team ID.
+
+```objective-c
+- (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions
+{
+  [Userdive setLogEnabled:YES];
+
+  // START DEVELOPER MODE
+  [Userdive startDeveloperMode:<your team id>];
+
+  return YES;
+}
+```
+
+
+### Capture images manually
+
+You need to capture the images of app view manually
+if you use custom view or 3rd party framework like a [Unity](https://unity3d.com/jp).
+
+Import USERDIVE iOS SDK header.
+
+```objective-c
+#import "Userdive.h"
+```
+
+Insert `[Userdive startDeveloperMode:<YOUR_TEAM_ID>]` into 
+`AppDeletegate:application:didFinishLaunchingWithOptions:` with your Team ID.
+
+```objective-c
+- (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions
+{
+  [Userdive setLogEnabled:YES];
+
+  // START DEVELOPER MODE WITHOUT TRACKERS
+  [Userdive startDeveloperMode:<your team id> trackers:nil];
+
+  return YES;
+}
+```
+
+When you want to capture view is displayed, call the capture method.
+
+Call `updateScreen` to prepare for capture the view.
+When you call `updateScreen` again, upload the captured image (You need to call `updateScreen` twice).
+
+```objective-c
+- (void)viewDidAppear:(BOOL)animated
+{
+  [super viewDidAppear:animated];
+
+  [Userdive updateScreen];
+  [Userdive setScreenName:@"screen A"];
+}
+```
+
+
+## Tracking gestures
+
+### Automatically detect view transitions
+
+USERDIVE iOS SDK can track user gesture and view transitions automatically.
+To start tracking, call `startTrackingMode:` when initializing SDK.
+
+Import USERDIVE iOS SDK header.
+
+```objective-c
+#import "Userdive.h"
+```
+
+Insert `[Userdive startTrackingMode:<YOUR_TEAM_ID>]` 
+`AppDeletegate:application:didFinishLaunchingWithOptions:` with your Team ID.
+
+```objective-c
+- (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions
+{
+    [Userdive setLogEnabled:YES];
+    [Userdive startTrackingMode:<your team id>];  // <-- HERE
+
+    return YES;
+}
+```
+
+### Manually view transitions
+
+In situations that have adopted a special view configuration
+(non-standard view placement and third-party libraries, etc.), 
+you might think that you want to switch the view tracking manually.
```

Comment:
"In the case of an adopted, special view configuration (nonstandard view placement and third-party libraries, etc.), you may want to consider switching to manual view transitions"